src/Stan/Inspection/Infinite.hs

```diff
@@ -35,29 +42,44 @@ infiniteInspectionsMap = fromList $ map (mapToFst inspectionId)
     [ stan0101
     , stan0102
     , stan0103
+    , stan0104
+    , stan0105
+    , stan0106
     ]
 
 -- | Smart constructor to create infinite 'Inspection'.
-mkInfiniteInspection :: Id Inspection -> NameMeta -> Inspection
-mkInfiniteInspection insId nameMeta@NameMeta{..} = Inspection
+mkInfiniteInspection :: Id Inspection -> NameMeta -> Pattern -> Inspection
+mkInfiniteInspection insId nameMeta@NameMeta{..} pat = Inspection
     { inspectionId = insId
     , inspectionName = "Infinite: " <> nameMetaPackage <> "/" <> nameMetaName
     , inspectionDescription =
         "Usage of the '" <> nameMetaName <> "' function that hangs on infinite lists"
     , inspectionSolution = []
     , inspectionCategory = Category.infinite :| [Category.list]
     , inspectionSeverity = Warning
-    , inspectionAnalysis = FindName nameMeta
+    , inspectionAnalysis = FindName nameMeta pat
     }
 
 -- | 'Inspection' for 'stan0101' — infinite 'reverse' @STAN-0101@.
 stan0101 :: Inspection
-stan0101 = mkInfiniteInspection (Id "STAN-0101") (mkBaseListMeta "reverse")
+stan0101 = mkInfiniteInspection (Id "STAN-0101") (mkBaseListMeta "reverse") PatternAnything
```

Comment:
In such cases I with Haskell supported default arguments...

Reply:
:crying_cat_face: